pkg/core/service.go

```diff
@@ -86,6 +93,105 @@ func newService(options CreateServiceOptions) v1alpha1.Service {
 			},
 		},
 	}
+
+	if len(options.Env) > 0 {
+		var envVars []core_v1.EnvVar
+		for _, env := range options.Env {
+			envEntry := strings.SplitN(env, "=", 2)
+			if len(envEntry) != 2 {
+				return v1alpha1.Service{}, errors.New(fmt.Sprintf("unable to parse '%v', environment variables must provided as 'name=value'", env))
+			}
+			envKey := strings.TrimSpace(envEntry[0])
+			envValue := strings.TrimSpace(envEntry[1])
+			if strings.HasPrefix(envValue, "valueFrom:") {
+				source := strings.Split(envValue, ":")
+				if len(source) < 3 {
+					return v1alpha1.Service{}, errors.New(fmt.Sprintf("unable to parse 'valueFrom' entry '%v', it should be provided as a secretKeyRef, configMapKeyRef, fieldRef or resourceFieldRef", env))
+				}
+				sourceType := strings.TrimSpace(source[1])
+				switch sourceType {
+				case "secretKeyRef":
```

Comment:
I can't see any documentation for these values. Am I looking in the right place? If we don't document them, users won't use them and we don't need to support them. ;-)

Reply:
Where would the docs go? In the `Long:` description for the command?

Reply:
Yes please!